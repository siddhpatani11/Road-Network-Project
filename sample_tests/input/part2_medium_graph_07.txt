31 163
0 1 45 G
0 5 28 G
0 23 5 B
1 3 34 G
1 4 12 B
1 5 23 R
1 7 43 R
1 9 48 B
1 15 44 B
1 18 38 B
1 23 25 G
1 24 37 B
1 25 35 R
1 26 39 B
1 27 26 G
2 3 20 R
2 5 49 B
2 11 30 G
2 12 7 R
2 13 12 G
2 25 41 R
2 28 43 B
4 3 13 R
4 6 49 R
4 8 10 R
4 10 25 G
4 12 13 B
4 22 38 G
5 0 33 G
5 3 8 R
5 7 39 B
5 9 41 G
5 15 16 R
5 19 35 R
6 0 8 B
6 10 24 R
6 15 31 B
6 16 4 B
6 18 45 R
6 23 2 G
6 26 20 R
7 2 48 G
7 6 26 R
7 17 11 R
7 21 25 R
7 25 5 G
7 26 21 R
8 5 30 R
8 6 46 G
8 12 51 G
8 15 19 B
8 21 11 R
8 28 51 R
9 6 41 R
9 15 11 B
9 22 39 G
9 23 47 R
9 26 12 R
10 2 29 G
10 3 7 R
10 4 4 R
10 12 10 G
10 16 27 B
10 17 1 R
10 20 21 G
10 23 3 R
10 26 7 R
10 27 45 G
11 2 42 G
11 3 41 R
11 9 14 R
11 16 25 B
11 18 31 G
11 19 47 B
11 27 12 B
12 11 49 B
12 21 32 R
13 1 34 B
13 4 9 R
13 11 26 B
13 17 30 B
13 19 6 R
13 30 10 G
14 7 29 G
14 9 26 G
14 19 17 R
14 28 36 R
15 4 16 R
15 7 40 B
15 27 35 B
16 1 25 B
16 8 23 G
16 11 31 B
16 15 7 G
16 22 25 G
16 27 6 R
17 16 22 B
17 18 35 G
17 28 20 R
18 0 3 G
18 2 31 R
18 20 13 G
18 27 15 B
18 28 47 R
18 30 32 B
19 0 6 G
19 3 17 B
19 8 36 B
19 9 33 B
19 13 42 B
19 17 41 B
19 22 43 R
19 28 3 R
20 8 43 R
20 11 27 G
20 15 45 G
20 16 21 R
21 0 13 B
22 3 25 R
22 6 19 G
22 9 15 B
22 12 34 R
22 14 9 G
22 21 14 R
23 1 23 R
23 6 28 G
23 9 41 R
23 14 35 B
23 15 7 G
24 0 33 G
24 7 20 B
24 14 35 G
24 21 50 R
24 30 22 B
25 0 14 G
25 1 7 G
25 3 4 G
25 5 15 R
25 17 50 B
25 20 28 G
25 26 36 R
26 3 26 G
26 18 26 G
26 22 50 G
26 28 46 B
27 1 37 B
27 21 9 G
28 0 25 G
28 10 21 G
28 20 39 B
28 22 3 R
28 23 5 R
28 25 49 G
28 26 43 G
29 1 42 G
29 5 23 G
29 10 19 G
29 16 26 R
29 25 16 R
30 5 29 G
30 6 12 G
30 21 4 R
30 27 18 B
