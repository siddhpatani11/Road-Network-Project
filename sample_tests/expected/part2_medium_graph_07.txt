23 G 36 R 23 R 177 R 57 G 75 R 109 G 28 G 47 R 150 R 41 B 92 R 227 R 201 R 209 B 35 G 7 G 32 G 58 G 93 B 52 R 149 G 47 R 80 - 0 R 165 B 120 G 68 G 38 B 71 - -1 G 125 
